databaseconnect.py

```diff
@@ -59,7 +72,10 @@ def setup_database():
     cur.execute(
         "CREATE TABLE IF NOT EXISTS directions_table(id INTEGER PRIMARY KEY AUTO_INCREMENT, origin_location VARCHAR(100), destination_location VARCHAR(100))"  # noqa: E501
     )
-    return db
+
+    cur.execute("SHOW TABLES")
+    TABLES = cur.fetchall()
+    return TABLES
```

Comment:
For `test_setup_database()` in `test_databaseconnect.py` to check whether tables were created as expected or not.